changelog.d/20220224_133151_jake_globus_collection.rst

```diff
@@ -0,0 +1,19 @@
+.. A new scriv changelog fragment.
+..
+.. Uncomment the header that is right (remove the leading dots).
+..
+.. Features
+.. --------
+..
+.. - A bullet item for the Features category.
+..
+.. Bugfixes
+.. --------
+..
+.. - A bullet item for the Bugfixes category.
+..
+.. Documentation
+.. -------------
+..
+.. - A bullet item for the Documentation category.
+..
```

Comment:
The leading `.. ` acts as a single-line comment. Please remove the comment lines and uncomment this section so it looks something like:

```rst
Documentation
-------------

- Document the ``globus-collection`` schema format.
```